Zend/zend_objects.c

```diff
@@ -325,7 +382,12 @@ ZEND_API zend_object *zend_objects_clone_obj(zend_object *old_object)
 		} while (p != end);
 	}
 
-	zend_objects_clone_members(new_object, old_object);
+	zend_objects_clone_members_ex(new_object, old_object, scope, properties);
 
 	return new_object;
 }
+
+ZEND_API zend_object *zend_objects_clone_obj(zend_object *old_object)
+{
+	return zend_objects_clone_obj_with(old_object, old_object->ce, &zend_empty_array);
```

Comment:
Nit: It seems like `scope` is unused when the properties array is empty. Maybe it's better to pass `NULL` here? Same in `zend_objects_clone_members()`.